### DISCUSSION TITLE
Kategorisér linjene etter retning

### DISCUSSION THREAD

**Discussion by forteller** [entur/tavla#561] [Ideas]
Når man skal velge hvilke linjer/ruter man vil vise i tavla fra en bestemt holdeplass, så er det gjerne snakk om to mulige varianter, den ene retningen eller den andre, østgånde eller vestgående feks. Noen ganger vil man gjerne bare vise den ene retningen eller den andre på tavla, og da hadde det vært fint om linjene var kategorisert etter det, så man kan slå av og på alle i én sleng, slik som Finn.no har at du kan velge fylke og så velge by under der igjen etterpå.

![Screenshot from 2021-12-23 23-13-42](https://user-images.githubusercontent.com/343561/147297067-98483226-9201-4618-af99-37583173972f.png)

For noen ganger er det ganske mange linjer, og det hadde vært mer oversiktlig om de var delt inn etter retning, tenker jeg.

**Comment by lindtvedtsebastian** [comment]
This discussion is linked to the old (deprecated) version of Tavla and will therefore be closed.
